packages/mesos-metrics-module/buildinfo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "requires": ["mesos", "boost-libs", "avro-cpp"],
+  "single_source" : {
+    "kind": "git",
+    "git": "git@github.com:mesosphere/dcos-stats.git",
```

Comment:
We haven't touched this yet, still redirects but definitly needs to change soon.